sdx-encrypted-submitter_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// Note these are integration tests that assume that the sdx-encrypted-submitter has been installed
+
+func TestNotSupplyingAnyArgument(t *testing.T) {
+
+	cmd := exec.Command("sdx-encrypted-submitter", "-e","something.txt", "-s", "somethingelse.txt")
```

Comment:
can use go run in this command so the test doesn't depend on the code being pre-compiled

Reply:
Done